### ISSUE TITLE
random crash

### ISSUE THREAD

**Issue by zouter** [dynverse/ti_monocle_ddrtree#3]
@kverstae noted that monocle ddrtree just randomly crashes for no apparent reason with a dataset of ~1000 cells and 20000 genes

Was able to reproduce:

```
library(dyno)

counts <- matrix(runif(1000*20000), nrow = 1000, ncol = 20000)
rownames(counts) <- seq_len(nrow(counts))
colnames(counts) <- seq_len(ncol(counts))

dataset <- wrap_expression(expression = counts, counts = counts)

trajectory <- infer_trajectory(dataset, "monocle_ddrtree")
```

Possibly related to dynverse/dyno#41

**Comment by zouter** [comment]
Apparently, this is just R cutting of the error message.

:notes: Just R things :notes:

**Comment by zouter** [comment]
Fixed with dynverse/dynwrap#142
